src/main/java/components/TopNavigation.java

```diff
@@ -0,0 +1,43 @@
+package components;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+
+import java.util.List;
+
+import static utils.WaitUtils.waitUntilElementsIsPresented;
+
+public class TopNavigation {
+    private final By menuBarList = By.cssSelector(".navigation li.level0");
+    private final By caretIconTopNavigationBar = By.cssSelector(".ui-menu-icon.ui-icon.ui-icon-carat-1-e");
```

Comment:
It's not a unique selector. It returns 8 elements. 
Should be unique and point to caret from menu.